templates/angular/igx-ts/custom-templates/tree-grid-finjs/files/src/app/__path__/localData.ts

```diff
@@ -0,0 +1,344 @@
+const lastYear = new Date().getFullYear() - 1;
+export const employeesData = [
+{
+  'EmployeeID': '56250fa57ab1535722e564a6',
```

Comment:
Lint doesn't like these properties being quoted, as they're a single word - you can remove the quotes from the property names and it still works fine.